### DISCUSSION TITLE
[Question] How different is KDE Neon from Kubuntu?

### DISCUSSION THREAD

**Discussion by Anonymo** [Sithuk/ubuntu-server-zfsbootmenu#35] [General]
I like Kubuntu but I don't like not being able to use the lastest KDE release.

**Comment by tripleo1** [comment]
TuxedoOS?

**Reply by Anonymo** [reply]
TuxedoOS runs ZFS + zfsbootmenu?

**Comment by online7878** [comment]
Hello, I am interested to install KDE neon instead of Kubuntu. I actually tried to install ubuntu server and then install KDE neon as single GUI on top. So I chrooted after finished script into mnt/ub_server and added the kde neon repository and did this:

wget -qO - 'https://archive.neon.kde.org/public.key' | sudo apt-key add -
cat <<EOF > /etc/apt/sources.list.d/neon.list
deb http://archive.neon.kde.org/user/ jammy main
deb-src http://archive.neon.kde.org/user/ jammy main
EOF

cat <<EOF > /etc/apt/preferences.d/99block-neon
Package: base-files
Pin: origin archive.neon.kde.org
Pin-Priority: 1
EOF

apt -y update
apt -y install neon-desktop

It started to install kde neon with this output:
________________________________________________________

 ┌─────────────┤ Pending kernel upgrade ├──────────────┐
 | Encfs security information                          │ 
 │                                                     │ 
 │ According to a security audit by Taylor Hornby      │ 
 │ (Defuse Security), the current implementation of    │ 
 │ Encfs is vulnerable or potentially vulnerable to    │ 
 │ multiple types of attacks. For example, an          │ 
 │ attacker with read/write access to encrypted data   │ 
 │ might lower the decryption complexity for           │ 
 │ subsequently encrypted data without this being      │ 
 │ noticed by a legitimate user, or might use timing   │ 
 │ analysis to deduce information.                     │ 
 │                                                     │ 
 │ Until these issues are resolved, encfs should not   │ 
 │ be considered a safe home for sensitive data in     │ 
 │ scenarios where such attacks are possible.          |    
 │                                                     │ 
 │                       <Ok>                          │ 
 │                                                     │ 
 └─────────────────────────────────────────────────────┘ 

 ┌─────────────┤ Pending kernel upgrade ├──────────────┐
 │                                                     │ 
 │ Newer kernel available                              │ 
 │                                                     │ 
 │ The currently running kernel version is             │ 
 │ 6.2.0-26-generic which is not the expected kernel   │ 
 │ version 6.2.0-39-generic.                           │ 
 │                                                     │ 
 │ Restarting the system to load the new kernel will   │ 
 │ not be handled automatically, so you should         │ 
 │ consider rebooting.                                 │ 
 │                                                     │ 
 │                       <Ok>                          │ 
 │                                                     │ 
 └─────────────────────────────────────────────────────┘ 

 ┌──────────┤ Outdated processor microcode ├───────────┐
 │                                                     │ 
 │ Processor microcode update                          │ 
 │                                                     │ 
 │ The currently running processor microcode revision  │ 
 │ is 0x0429 which is not the expected microcode       │ 
 │ revision 0x0430.                                    │ 
 │                                                     │ 
 │ Restarting the system to load the new processor     │ 
 │ microcode will not be handled automatically, so     │ 
 │ you should consider rebooting.                      │ 
 │                                                     │ 
 │                       <Ok>                          │ 
 │                                                     │ 
 └─────────────────────────────────────────────────────┘ 

No services need to be restarted.

No containers need to be restarted.

No user sessions are running outdated binaries.

No VM guests are running outdated hypervisor (qemu)
 binaries on this host.

Then I did:
update-initramfs -c -k all

exit
and rebooted but the ubuntu server with kde neon did not start, no boot partition was there. So what did I do wrong? Can anyone help me?

Thanks
